src/DocumentDbTests/Reading/Json/streaming_json_results.cs

```diff
@@ -28,15 +28,16 @@ public SimpleUser()
 
     public string ToJson()
     {
-        return $@"
-{{
-""Id"": ""{Id}"", ""Number"": {Number}, ""Address"":
-{{
-""Street"": ""{Address.Street}"", ""HouseNumber"": ""{Address.HouseNumber}""
-}},
-""UserName"": ""{UserName}"",
-""Birthdate"": ""{Birthdate.ToString("s")}""
-}}".Replace("\r\n", "").Replace("\n", "");
+        return $$"""
+{
```

Comment:
W/ raw string literals you could use intendantion too in order to make it more readable.